src/main/java/seedu/organizer/model/subtask/UniqueSubtaskList.java

```diff
@@ -129,4 +168,14 @@ protected DuplicateSubtaskException() {
         }
     }
 
+    //@@author natania
+    /**
+     * Makes the {@code Status} of all the subtasks in list not done.
+     */
+    public void makeAllSubtasksUndone(List<Subtask> subtasks) {
+        for (Subtask subtask : subtasks) {
+            ;
```

Comment:
Hey guys if the feature cannot be implemented in time, then perhaps it would be wise to remove this piece of code and update the user guide on the limitations.